vm/actor_interface/src/builtin/power/mod.rs

```diff
@@ -139,77 +106,131 @@ impl State {
         &self,
         s: &BS,
         miner: &Address,
-    ) -> Result<Option<Claim>, Box<dyn Error>> {
+    ) -> anyhow::Result<Option<Claim>> {
         match self {
-            State::V0(st) => Ok(st.miner_power(s, miner)?.map(From::from)),
-            State::V2(st) => Ok(st.miner_power(s, miner)?.map(From::from)),
-            State::V3(st) => Ok(st.miner_power(s, miner)?.map(From::from)),
-            State::V4(st) => Ok(st.miner_power(s, miner)?.map(From::from)),
-            State::V5(st) => Ok(st.miner_power(s, miner)?.map(From::from)),
-            State::V6(st) => Ok(st.miner_power(s, miner)?.map(From::from)),
+            State::V0(st) => {
+                let claim = st
+                    .miner_power(s, miner)
```

Comment:
how about a small macro to reduce code duplication?

Reply:
Having a consistent set of macros for dealing with all the different actor versions would be great. I'll leave it for a future PR, though.